toolchest_client/tools/api.py

```diff
@@ -388,6 +391,44 @@ def megahit(output_path=None, tool_args="", read_one=None, read_two=None, interl
     return output
 
 
+def python3(script, inputs=[], output_path=None, tool_args="", **kwargs):
+    """Runs Python3 via Toolchest.
+    NOTE: This tool is currently only available for customers with contracts
+    The script has a few requirements if inputs or outputs are expected. Inputs will be uploaded to `./input/` so the
+    script must use that prefix with hardcoded paths or when adding to tool_args to be parsed via sys.argv. Any output
+    that is expected to be returned needs to be stored at `./output/`. The directory will be compressed via pigz and
+    decompressed on the client.
+
+    :param script: Path to the python script to run
+    :param inputs: (optional) Path(s) to the input files to run with the python script. Will be uploaded to a directory
+    `./input/` for use in the on the instance.
+    :param output_path: (optional) Local path to where the output file(s) will be downloaded.
+    :param tool_args: (optional) Additional arguments to be passed to Python3. Should NOT specify the script here but
+    any other args after the script should be included.
```

Comment:
Rephrase with a positive example and/or a more clear example of what not to do, or just remove the added context.

(More specifically, modify or remove "Should NOT specify the script here but any other args after the script should be included.")